cmd/upgrade.go

```diff
@@ -10,7 +10,8 @@ import (
 )
 
 func init() {
-	upgrade.Flags().StringP("config", "c", "fogg.json", "Use this to override the fogg config file.")
+	//Defaults to fogg.yml but fogg.json can be used as an argument instead
```

Comment:
i feel like this comment is implied, so there might not be much value in keeping it